.github/workflows/ci.yaml

```
@@ -88,7 +88,11 @@ jobs:
             if [[ ${{ matrix.os }} == "macos-latest" ]]; then
               export PYOPENCL_TEST=port:pthread
             fi
-            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw .
+            # One test at a time to address ballooning memory.  See MIRGE-Com issue:
```

```suggestion
            # One test at a time to address ballooning memory.
```
(Super minor: That link describes itself, so no need to state what it is.)

---

Heh, I keep doing that.